src/lib/alignment.ts

```diff
@@ -0,0 +1,23 @@
+export interface Alignment {
```

Comment:
we should add word property (number) to interface and makeAlignment

Reply:
Done